doc/spire_server.md

```diff
@@ -78,6 +78,8 @@ This may be useful for templating configuration files, for example across differ
 | `profiling_freq`                    | Frequency of dumping profiling data to disk. Only enabled when `profiling_enabled` is `true` and `profiling_freq` > 0.                                                                                                                          |                                                                |
 | `profiling_names`                   | List of profile names that will be dumped to disk on each profiling tick, see [Profiling Names](#profiling-names)                                                                                                                               |                                                                |
 | `profiling_port`                    | Port number of the [net/http/pprof](https://pkg.go.dev/net/http/pprof) endpoint. Only used when `profiling_enabled` is `true`.                                                                                                                  |                                                                |
+| `prune_attested_nodes_expired_for`      | Enables periodic purging of attested node records with expired SVIDs where the expiry time further in the past than the specidied duration. Non-reattestable nodes are not pruned unless `prune_non_reattestable_nodes` is set to `true`. Banned nodes are not pruned. |
+| `prune_non_reattestable_nodes`          | Includes non-reattestable expired nodes into consideration for pruning                                                                                                                  | false                                                          |
```

Comment:
Consider renaming this to `prune_tofu_nodes` for clarity,
we're removing "TOFU" nodes, which allows the associated documents to be reused. (and it is important to mention than Banned nodes are not affected)